README.md

```diff
@@ -8,6 +8,17 @@ Basic Audit Log Patterns
 [(BALP)](https://profiles.ihe.net/ITI/BALP/volume-1.html#1-52-basic-audit-log-patterns) 
 implementation guide.
 
+## Table of Contents
```

Comment:
github generates its own ToC, just FYI

click the "bullet list" icon, right of the pencil icon. can't upload screenshot at the moment (bad wifi)

Reply:
Thanks, that's handy!

I still see value in it, given that I and others may have overlooked that feature!